hooks/02-spread-support.chroot

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh -x
+
```

Comment:
Maybe a comment here that this is really only required while developing? I'm a bit scared that this may be forgotten and leak at some point into the store :/

Reply:
Sure